earth_enterprise/src/fusion/autoingest/geAssetRoot.cpp

```diff
@@ -32,23 +32,39 @@ struct WantPerms {
 // ***  Special Directories
 // ****************************************************************************
 
-// must be in same order as SpecialDirs enum
+// Both special dirs structs must be in same order as SpecialDirs enum
+// These are the standard permissions
 const WantPerms special_dirs[] = {
     {"",           0755},
     {".config",    0777},
     {".state",     0755},
     {".userdata",  0777},
     {".privatedb", 0700}
 };
+
+// These are the secure permissions
+const WantPerms secure_special_dirs[] = {
+    {"",           0755},
+    {".config",    0775},
+    {".state",     0755},
+    {".userdata",  0775},
+    {".privatedb", 0700}
+};
+
```

Comment:
very minor: might be better to use `std::array`s

Reply:
The only reason I didn't change the data type is that it is supposed to match up to an enum defined in the header.

Reply:
If you think it would be worth it I can certainly change it though.